sphinxgallery/gen_gallery.py

```diff
@@ -46,12 +48,14 @@ def generate_gallery_rst(app):
     if not plot_gallery:
         return
 
-    build_image_dir = os.path.join(app.builder.outdir, '_images')
-    clean_gallery_out(build_image_dir)
+    #clean_gallery_out(app.builder.outdir)
```

Comment:
was that commented out on purpose, i.e. don't we want to remove the 'sphx-glr*.png' from build/_images?